client/src/components/molecules/InlineComments.jest.spec.js

```diff
@@ -275,4 +275,11 @@ describe("InlineComments", () => {
     await items.at(2).find('[data-cy="collapseRepliesButton"]').trigger("click")
     expect(findReplies(items.at(2))).toHaveLength(10)
   })
+
+  test("scroll to top button triggers scrolling functionality when clicked upon", async () => {
+    const { wrapper } = wrapperFactory()
+    const button = wrapper.findComponent({ ref: "scroll_to_top_button" })
+    await button.trigger("click")
+    expect(wrapper.emitted("scrollInlineCommentsTop")).toHaveLength(1)
+  })
```

Comment:
```suggestion
```

I think the better way to test this would be to mock out the `scroll` module and `setVerticalScrollPosition` and then assert that they've been called with the appropriate element as their target.  That said, I don't think that effort is worth it in this case.  Testing an event emitted only for testing however creates a false sense of the functionality having been tested.  

If a parent component were listening for this event and dependent on it to perform some other action, then for sure, this would be a worthwhile test.

Given how much is reliant on a browser here, a cypress test may make more sense (visit page, get button, click, assert that the first comment's text is visible again) (a quick look at the docs suggests that should(be.visible) does take into account if the element is actually in the current scroll window).  Again though, I don't think going out of our way to write that test is worth the effort.